Misc/NEWS.d/next/Library/2021-07-20-18-34-16.bpo-44353.ATuYq4.rst

```diff
@@ -0,0 +1,2 @@
+Add helper ``functools._idfunc`` to make ``typing.NewType`` ``__call__``
```

Comment:
Not sure we should even mention where the helper is located. We can just say that we made `NewType.__call__` faster by implementing it in C.

Reply:
Agree, changed.